Makefile

```diff
@@ -74,141 +110,3 @@ check-node-and-npm-version-parity:
 			exit 1; \
 		} \
 	fi;
-
-# Builds Calypso (desktop)
-build: check-node-and-npm-version-parity install
-	@echo "Building Calypso (Desktop on branch $(RED)$(CALYPSO_BRANCH)$(RESET))"
-	@cd calypso && CALYPSO_ENV=desktop MINIFY_JS=false npm run build
-	@rm -f $(THIS_DIR)/calypso/public/devmodules.*
-
-build-if-not-exists:
-	@if [ -f $(CALYPSO_JS) ]; then true; else make build; fi
-
-build-if-changed: build-if-not-exists
-	@if [ $(CALYPSO_CHANGES_STD) -eq 0 ]; then true; else make build; fi;
```

Comment:
this and `build-if-not-exists` are already built-in to `make`, so they are slow and redundant here :smile: